src/blocks/pricing-table/controls.js

```diff
@@ -57,7 +57,13 @@ class Controls extends Component {
 						value={ contentAlign }
 						onChange={ ( nextContentAlign ) => setAttributes( { contentAlign: nextContentAlign } ) }
 					/>
-					<Toolbar
+          {
+            /**
+             * Using ToolbarGroup to make controls render.
+             * See https://github.com/godaddy-wordpress/coblocks/issues/1670
+             */
+          }
+					<ToolbarGroup
```

Comment:
This same fix seems to have been applied upstream in https://github.com/godaddy-wordpress/coblocks/pull/1671. I don't think we need a comment here.
```suggestion
					<ToolbarGroup
```